toolkit/tools/graphpkgfetcher/graphpkgfetcher.go

```diff
@@ -57,12 +57,30 @@ func main() {
 	logger.InitBestEffort(*logFile, *logLevel)
 
 	dependencyGraph := pkggraph.NewPkgGraph()
+	oldCachedGraph := pkggraph.NewPkgGraph()
 
 	err := pkggraph.ReadDOTGraphFile(dependencyGraph, *inputGraph)
 	if err != nil {
 		logger.Log.Panicf("Failed to read graph to file. Error: %s", err)
 	}
 
+	if _, err := os.Stat(*outputGraph); err == nil {
```

Comment:
Could this be its own function, and could you explain the effects this has on the resulting graph?

I'm assuming this is to avoid scanning for what package might provide a given requirement.

Will this cause the fetcher to not retrieve the actual rpm? (Say we delete the .rpm from the cache, but the old graph .dot file is still there).